src/modules/rlm_totp/rlm_totp.c

```diff
@@ -186,7 +186,8 @@ module_rlm_t rlm_totp = {
 		.instantiate	= mod_instantiate
 	},
 	.method_names = (module_method_name_t[]){
-		{ .name1 = "authenticate",	.name2 = CF_IDENT_ANY,		.method = mod_authenticate,	.method_env = &method_env },
+		{ .name1 = "authenticate",	.name2 = CF_IDENT_ANY,	.method = mod_authenticate,	.method_env = &method_env },
+		{ .name1 = CF_IDENT_ANY,	.name2 = CF_IDENT_ANY,	.method = mod_authenticate,	.method_env = &method_env },
```

Comment:
I'm not clear why this is necessary?  The totp module really should only be authenticating users.

I guess that it _could_ go anywhere to authenticate users, but we generally put the `pap`, `chap`, etc. authentications into an `authenticate` section